packages/video_player/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: video_player
 description: Flutter plugin for displaying inline video with other Flutter
   widgets on Android and iOS.
 author: Flutter Team <flutter-dev@googlegroups.com>
-version: 0.8.0
+version: 0.8.1
```

Comment:
Should this be a 0.9.0 bump?  Could we end up surprising anyone with this change if they had some other work around in place to handle it?

Reply:
Agree. Updated to 0.9.0